pyaedt/edb_core/EDB_Data.py

```diff
@@ -1149,7 +1198,7 @@ def position(self):
         Returns
         -------
         list
-            List of the pin position in the format ``[x, y]``.
+            List of ``[x. y]``` coordinates for the pin position.
```

Comment:
```suggestion
            List of ``[x, y]``` coordinates for the pin position.
```